tests/activity-report.spec.ts

```diff
@@ -0,0 +1,159 @@
+    await page.getByRole('group', { name: 'Reason for activity' }).getByTestId('label').locator('div').filter({ hasText: '- Select -' }).nth(2).click();
+    await page.locator('#react-select-9-option-0').click();
+    await blur(page);
+    await page.getByLabel('Start date *mm/dd/yyyy').click();
+    await page.getByLabel('Start date *mm/dd/yyyy').fill('12/01/2020');
+    await page.getByLabel('End date *mm/dd/yyyy').click();
+    await page.getByLabel('End date *mm/dd/yyyy').fill('12/01/2050');
+    await page.getByLabel('Duration in hours (round to the nearest half hour) *').click();
+    await page.getByLabel('Duration in hours (round to the nearest half hour) *').fill('5');
+    await page.getByRole('group', { name: 'What TTA was provided *' }).getByText('Training').click();
+    await page.getByText('Virtual').click();
+    await page.getByText('Video').click();
+    await page.locator('#participants div').filter({ hasText: '- Select -' }).nth(1).click();
+    await page.locator('#react-select-11-option-0').click();
+    await blur(page);
+    await page.getByLabel('Number of participants involved *').click();
+    await page.getByLabel('Number of participants involved *').fill('5');
+    await page.getByRole('button', { name: 'Save and continue' }).click();
+
+    // create the first goal
+    await page.getByTestId('label').locator('div').filter({ hasText: '- Select -' }).nth(2).click();
+    await page.locator('#react-select-15-option-0').getByText('Create new goal').click();
+    await page.getByTestId('textarea').click();
+    await page.getByTestId('textarea').fill('g1');
+    await page.getByText('Yes').click();
+    await page.getByRole('button', { name: 'Save goal' }).click();
+    await page.locator('.css-125guah-control > .css-g1d714-ValueContainer').click();
+    await page.keyboard.press('Enter');
+    await page.getByLabel('TTA objective *').click();
+    await page.getByLabel('TTA objective *').fill('g1o1');
+    await page.locator('.css-125guah-control > .css-g1d714-ValueContainer').click();
+    await page.locator('#react-select-21-option-0').click();
+    await blur(page);
+    await page.getByRole('textbox', { name: 'TTA provided for objective' }).locator('div').nth(2).click();
+    await page.keyboard.type('hello');
+    await page.getByRole('button', { name: 'Save goal' }).click();
+
+    // extract the AR number from the URL:
+    const url = await page.url();
+    const arNumber = url.split('/').find((part) => /^\d+$/.test(part));
+
+    // create the second goal
+    await page.getByRole('button', { name: 'Add new goal' }).click();
+    await page.getByTestId('textarea').click();
+    await page.getByTestId('textarea').fill('g2');
+    await page.getByRole('group', { name: 'Is this a Recipient TTA Plan Agreement (RTTAPA) goal?*' }).getByText('Yes').click();
+    await page.locator('.css-125guah-control > .css-g1d714-ValueContainer').click();
+    await page.locator('#react-select-25-option-0').click();
```

Comment:
I assume this test works consistently? I have (in the past) struggled to get the same output every time from react-select without using the props to specify ids, etc.

Reply:
it passes consistently as long as the styling for the selector stays the same. this particular selector (and others within the test that have similar hashes) is nested within the select component, so changing the padding/margin of the parent element should not change this hash here

Reply:
Although I am mystified by the reference to margin and padding, I think what you are saying is that it seems to use the same hash every time. I accept! 

I think that it might have been in JestDOM where Emotion was generating different hashes to prefix the CSS each test run (css-_g1d714_ would become css-_125guah_ or whatever)... but I guess if that does happen happens, we'll know to adjust the code here, so it's a moot point